frontend/src/components/AllPosts.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import PostsList from './PostsList'
+
+const AllPosts = () => {
+  const posts = useSelector(({ posts }) => posts?.data)
```

Comment:
Maybe we could start extracting those selectors into methods like `getPosts()`. Can be useful later on in case of more complex selectors 👯